package.json

```diff
@@ -14,6 +14,7 @@
     "url": "https://github.com/unexpectedjs/unexpected"
   },
   "scripts": {
+    "prepare": "make build/lib",
```

Comment:
:ok_hand:

This has bothered me a lot when I've installed unexpected via a git dep.

Reply:
This has been folded into https://github.com/unexpectedjs/unexpected/pull/683.y